packages/components/breadcrumb/_breadcrumb.scss

```diff
@@ -37,12 +37,13 @@
 
   &:not(:last-child):after {
     background: url("data:image/svg+xml,%3Csvg class='nhsuk-icon nhsuk-icon__chevron-right' xmlns='http://www.w3.org/2000/svg' fill='%23aeb7bd' height='18' width='18' viewBox='0 0 24 24' aria-hidden='true'%3E%3Cpath d='M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z'%3E%3C/path%3E%3C/svg%3E")
-      right 0 top 4px no-repeat;
+      right 0 top 0 no-repeat;
     content: "";
     display: inline-block;
-    height: 18px;
+    height: 19px;
     margin-left: 10px;
     margin-right: 2px;
+    vertical-align: middle;
     width: 18px;
```

Comment:
I'm not entirely sure what's going on here - but if you're happy it works, great.

Perhaps the entire section and then each magic number could have a comment describing the purpose?

Reply:
Added a comment